app/main/authorize/access_token_sign_in_required.py

```diff
@@ -40,44 +40,39 @@ def protected_route():
     def decorated_view(*args, **kwargs):
         g.access_token_sign_in_required = True  # Set attribute on g
 
+        access_token = session.get("access_token")
```

Comment:
the only change in this file is the removal of the try except. We dont need to set `access_token_sign_in_required` bavk to False at the end of the request since flask's global object `g` gets reset on each request, so I just simplified this.